oshi-core/src/main/java/oshi/util/ConfigUtil.java

```diff
@@ -0,0 +1,164 @@
+            if (in != null)
+                configuration.load(in);
+            else
+                LOG.warn("Default configuration not found");
+        } catch (IOException e) {
+            LOG.warn("Failed to load default configuration");
+        }
+    }
+
+    /**
+     * Get the {@code String} property associated with the given key.
+     * 
+     * @param key The property key
+     * @return The property value or {@code null} if not found
+     */
+    public static String get(String key) {
+        return configuration.getProperty(key);
+    }
+
+    /**
+     * Get the {@code String} property associated with the given key.
+     * 
+     * @param key The property key
+     * @param def The default value
+     * @return The property value or the default if not found
+     */
+    public static String get(String key, String def) {
+        return configuration.getProperty(key, def);
+    }
+
+    /**
+     * Get the {@code Integer} property associated with the given key.
+     * 
+     * @param key The property key
+     * @return The property value or {@code 0} if not found
+     */
+    public static int getInteger(String key) {
+        return getInteger(key, 0);
+    }
+
+    /**
+     * Get the {@code Integer} property associated with the given key.
+     * 
+     * @param key The property key
+     * @param def The default value
+     * @return The property value or the default if not found
+     */
+    public static int getInteger(String key, int def) {
+        String value = configuration.getProperty(key);
+        return value == null ? def : Integer.parseInt(value);
+    }
+
+    /**
+     * Get the {@code Double} property associated with the given key.
+     * 
+     * @param key The property key
+     * @return The property value or {@code 0.0} if not found
+     */
+    public static Double getDouble(String key) {
+        return getDouble(key, 0.0);
+    }
+
+    /**
+     * Get the {@code Double} property associated with the given key.
+     * 
+     * @param key The property key
+     * @param def The default value
+     * @return The property value or the default if not found
+     */
+    public static Double getDouble(String key, double def) {
+        String value = configuration.getProperty(key);
+        return value == null ? def : Double.parseDouble(value);
+    }
+
+    /**
+     * Get the {@code Boolean} property associated with the given key.
+     * 
+     * @param key The property key
+     * @return The property value or {@code false} if not found
+     */
+    public static boolean getBoolean(String key) {
+        return getBoolean(key, false);
+    }
+
+    /**
+     * Get the {@code Boolean} property associated with the given key.
+     * 
+     * @param key The property key
+     * @param def The default value
+     * @return The property value or the default if not found
+     */
+    public static boolean getBoolean(String key, boolean def) {
```

Comment:
Since the `String` getter doesn't include the type, why not just use `get()` here as well?